examples/attaching-middleware/test/test.js

```diff
@@ -0,0 +1,18 @@
+const {
+  decorators: { test },
+} = require('../boltzmann')
+const tap = require('tap')
+
+const _ = test({})
+
+tap.test(
+  'it works!',
+  _(async (assert) => {
+    const response = await assert.request({
+      method: 'GET',
+      url: '/hello/world',
+    })
+
+    assert.equal(response.payload, 'hello world')
```

Comment:
Running the test locally, I can see the 'hello world' response get swallowed and return a 204 causing this assert to fail. I'm trying to track down why...